packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react'
+import PropTypes from 'react-proptypes'
+
+import { withMutations } from 'cozy-client'
+
+import AccountCreator from './AccountCreator'
+import AccountEditor from './AccountEditor'
+import TriggerSuccessMessage from './TriggerSuccessMessage'
+import { triggersMutations } from '../connections/triggers'
+import {
+  buildKonnectorCron,
+  buildKonnectorTriggerAttributes
+} from '../helpers/triggers'
+
+const IDLE = 'IDLE'
+const RUNNING = 'RUNNING'
+const LOGGED = 'LOGGED'
+const SUCCESS = 'SUCCESS'
+
+/**
+ * Deals with konnector configuration, i.e encapsulate account creation or
+ * edition
+ * @type {Component}
+ */
+export class TriggerManager extends Component {
+  state = {
+    status: IDLE
+  }
+
+  /**
+   * Creation/update start handler
+   * Set the status to RUNNING as soon as an account is being created
+   * or updated.
+   */
+  handleAccountMutation = () =>
+    this.setState({
+      status: RUNNING
+    })
+
+  /**
+   * Account creation success handler
+   * @param  {Object}  account Created io.cozy.accounts document
+   * @return {Object}          io.cozy.jobs document, runned with account data
+   */
+  handleAccountCreationSuccess = async account => {
+    const { createTrigger, konnector } = this.props
+
+    const trigger = await createTrigger(
+      buildKonnectorTriggerAttributes({
```

Comment:
Nit: I found having namespaces leads to shorter function names : `triggers.buildForKonnector`